components/script/dom/webidls/HTMLCollection.webidl

```diff
@@ -4,7 +4,6 @@
 
 // https://dom.spec.whatwg.org/#interface-htmlcollection
 
-[LegacyUnenumerableNamedProperties]
```

Comment:
Because I fumbled when I introduced this. They are already unenumerable in HTMLCollection. When I upstreamed it I added (or someone else, I don't remember) a check that looks at ancestor interfaces to see if they are already unenumerable there, in which case derived interfaces shouldn't mention it.